package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hapi-hydra-version",
-  "version": "1.0.3",
-  "description": "versioning for hapi, using all three methods: https://www.troyhunt.com/your-api-versioning-is-wrong-which-is/",
+  "version": "1.0.4",
+  "description": "versioning for happy, using all three methods: https://www.troyhunt.com/your-api-versioning-is-wrong-which-is/",
```

Comment:
happy?

Reply:
oops.  I had fixed that in another commit. Looks like it was lost in a merge. thanks :)